src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -81,6 +82,17 @@ public EditPersonDescriptorBuilder withAffiliations(String... tags) {
         return this;
     }
 
+    /**
+     * Parses the {@code affiliationHistory} into a {@code Set<Affiliation>} and
+     * set it to the {@code EditPersonDescriptor}
+     * @return EditPersonDescriptorBuilder
+     */
+    public EditPersonDescriptorBuilder withAffiliationHistory(String... tags) {
```

Comment:
Maybe using `affiliations` as the parameter name is better here. Likewise for the function above (javadoc as well)! Might have been missed out in previous PRs.

Reply:
good catch, thanks!